src/Fuzz.elm

```diff
@@ -241,16 +241,25 @@ string =
         Shrink.string
 
 
+{-| A fuzzer for string values. Generates random printable ascii strings whose
+length can be specified.
+-}
+stringOfLength : Int -> Int -> Fuzzer String
```

Comment:
an example in the docs would go a really long way - e.g, what does `stringOfLength 5 5` return? what about error catching, what does `stringOfLength 5 0` do?